routing/http/contentrouter/contentrouter.go

```diff
@@ -300,3 +307,45 @@ func (c *contentRouter) SearchValue(ctx context.Context, key string, opts ...rou
 
 	return ch, nil
 }
+
+func (c *contentRouter) GetClosestPeers(ctx context.Context, pid peer.ID) (<-chan peer.AddrInfo, error) {
+	iter, err := c.client.GetClosestPeers(ctx, pid)
+	if err != nil {
+		return nil, err
+	}
+	defer iter.Close()
```

Comment:
This needs to be done in the goroutine, otherwise the function returns and the iter closes before the goroutine even has time to run.